src/styles/_catalog.scss

```diff
@@ -16,7 +16,21 @@
     margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 768px) and (max-width: 1024px) {
+    .catalog {
+      grid-template-columns: repeat( 3, 200px);
+    }
+  }
```

Comment:
you already write query in `.catalog` selector. it's not working 
![image](https://user-images.githubusercontent.com/22824431/231136902-a2cf0620-eba7-4e63-9883-fe0129d1739a.png)
```suggestion
  @media (min-width: 768px) and (max-width: 1024px) {
    grid-template-columns: repeat( 3, 200px);
  }
```